pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -130,6 +131,14 @@ def test_loc_multiindex(self):
             with pytest.raises(KeyError, match=r"^2L?$"):
                 mi_int.ix[2]
 
+        s = Series(range(8), index=MultiIndex.from_product(
```

Comment:
can you make this another test name. add the issue number as a comment

Reply:
sorry i dont understand, should i add this to a new function?

Reply:
can you make the name more informative, maybe

test_loc_multiindex_too_many_dims